src/util/ConfigManager/ConfigManager.h

```diff
@@ -354,11 +370,31 @@ class ConfigManager {
   }
 
   /*
-  @brief Provide a range of tuples, that hold references to the key value pairs
-  in `configurationOptions_`, but with the pointer dereferenced.
+  @brief A vector to all the configuratio options, held by this manager,
+  represented with their json paths and reference to them. Options held by a sub
+  manager, are also included with the path to the sub manager as prefix.
+
+  @param pathPrefix This prefix will be added to all configuration option json
+  paths, that will be returned.
+  */
+  std::vector<std::pair<std::string, ConfigOption*>> configurationOptions(
+      std::string_view pathPrefix = "");
+  std::vector<std::pair<std::string, const ConfigOption*>> configurationOptions(
+      std::string_view pathPrefix = "") const;
```

Comment:
Are those ever called with arguments that are not `""` (might be that I missed this).

Reply:
(Maybe the second argument is needed for the Impl, but not for the actual member function).

Reply:
Ah, I think this actually an artifact. Can't quite remember, which use there were going to have though. Got rid of them in [c4893a6](https://github.com/ad-freiburg/qlever/pull/1071/commits/c4893a6bd1ff1467d72a54d61dcf4d676fbd5cd8).